packages/material-ui-styles/src/makeStyles/makeStyles.d.ts

```diff
@@ -3,9 +3,11 @@ import {
   ClassNameMap,
   PropsOfStyles,
   Styles,
-  ThemeOfStyles,
   WithStylesOptions,
+  StyleRules,
+  StyleRulesCallback,
 } from '@material-ui/styles/withStyles';
+import { Theme } from '@material-ui/core';
```

Comment:
That's not true though. `@material-ui/styles` has no dependency on the core. It's the other way around. The default should be `unknown`.

Reply:
Got it, will remove the reference to core.